libs/dh/message-archive/feature-log-search/src/lib/searchresult/dh-message-archive-log-search-result.component.html

```diff
@@ -36,14 +36,24 @@
     <!-- sender gln -->
     <ng-container matColumnDef="sender">
       <th mat-header-cell *matHeaderCellDef>{{ transloco("sender") }}</th>
-      <td mat-cell *matCellDef="let element">{{ element.senderGln }}</td>
+      <td mat-cell *matCellDef="let element">
+        <container-element [ngSwitch]="element.httpData">
+          <label *ngSwitchCase="'request'" type="info">{{
```

Comment:
suggestion: The `label` element is usually used in relation to other elements like `input`, `textarea`, etc. I can see in the rest of the file that the `watt-badge` component was used for a similar case. Could it be that `watt-badge` should be used here too?

Reply:
no badge here it should only be text

Reply:
Then unless you need specific styling, you can replace `label` with `ng-container` and remove the `type` property. `ng-container` is a generic element in Angular that will not render the element in the HTML DOM tree.